<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/resources/img/bg-home.jpg&quot;);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>381</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-size: 35px;
	font-weight: 1400;
background:none;</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: transparent;
	border: none;
	border-bottom: 1px solid black;
	font-size: 15px;</string>
    </property>
    <property name="placeholderText">
     <string>Full name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_email">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: transparent;
	border: none;
	border-bottom: 1px solid black;
	font-size: 15px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_password">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: transparent;
	border: none;
	border-bottom: 1px solid black;
	font-size: 15px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Choose Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_password2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	background: transparent;
	border: none;
	border-bottom: 1px solid black;
	font-size: 15px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	font-size: 13px;
	font-weight: 500;
background:none;</string>
    </property>
    <property name="text">
     <string>I agree to terms and conditions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	color: white;
	font-weight: 700;
	border-radius: 4px;
	background: green;</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
color: red;
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>20</y>
      <width>831</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
	background: rgba(255,255, 255,0.5);
	border-radius: 5;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:grey;</string>
     </property>
     <property name="text">
      <string>&lt;---</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="male">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>241</y>
      <width>54</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>male</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="female">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>241</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;</string>
    </property>
    <property name="text">
     <string>female</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>284</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-size: 15px;

	background: rgba(255,255, 255,0.2);
	border-radius: 5;

</string>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>285</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;
font-weight:200;
background:none;
</string>
    </property>
    <property name="text">
     <string>DD/MM/YYYY :</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>user_name</zorder>
   <zorder>user_email</zorder>
   <zorder>user_password</zorder>
   <zorder>pushButton</zorder>
   <zorder>warning</zorder>
   <zorder>female</zorder>
   <zorder>checkBox</zorder>
   <zorder>dateEdit</zorder>
   <zorder>male</zorder>
   <zorder>label_pic</zorder>
   <zorder>label_2</zorder>
   <zorder>user_password2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
